runtime/doc/lua.txt

```diff
@@ -18,7 +18,7 @@ an idea of what lurks beneath: >
 
 Nvim includes a "standard library" |lua-stdlib| for Lua.  It complements the
 "editor stdlib" (|builtin-functions| and Ex commands) and the |API|, all of
-which can be used from Lua code. A good overview of using Lua in neovim is
+which can be used from Lua code.  A good overview of using Lua in neovim is
```

Comment:
One space between sentences seems more common in `lua.txt`.

Reply:
But then let's be consistent, The sentences right before and after have two spaces. Personally I prefer one, as we all write like that online and most media seems to as well. Besides, any char gained matters in these short lines.

Reply:
If you touch these sections, it would be appreciated if you took the opportunity to improve them otherwise.

Double-spacing is a relic of the typewriter past that needs to die already.

Reply:
Ok, I've removed all double spacing.